lib/modules/manager/gradle/parser.ts

```diff
@@ -396,6 +397,19 @@ function processLibraryDep(input: SyntaxHandlerInput): SyntaxHandlerOutput {
   return res;
 }
 
+function processApplyFrom({
+  tokenMap,
+  variables,
+}: SyntaxHandlerInput): SyntaxHandlerOutput {
+  let scriptFile: string | null = tokenMap.scriptFile?.value;
+  if (tokenMap.scriptFile?.type === TokenType.StringInterpolation) {
```

Comment:
@zharinov we should optimize types, so typescript can narrow it here. can be done in separate PR

Reply:
Can be a rabbit hole that would cost us days of work